server/routerlicious/packages/lambdas-driver/src/document-router/documentPartition.ts

```diff
@@ -100,7 +100,12 @@ export class DocumentPartition {
 			return;
 		}
 
-		void this.q.push(message);
+		this.q.push(message).catch((error) => {
+			const lumberjackProperties = {
+				...getLumberBaseProperties(this.documentId, this.tenantId),
+			};
+			Lumberjack.error("Error pushing raw message to queue", lumberjackProperties, error);
```

Comment:
nit: to tell this log apart from the one below in partition.ts, maybe we can slightly rephrase the error message to:

`"Error pushing raw message to queue in documentPartition"`